test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -139,13 +139,16 @@ protected final RestClient adminClient() {
     }
 
     private void wipeCluster() throws IOException {
-        // wipe indices
-        try {
-            adminClient().performRequest("DELETE", "*");
-        } catch (ResponseException e) {
-            // 404 here just means we had no indexes
-            if (e.getResponse().getStatusLine().getStatusCode() != 404) {
-                throw e;
+        final boolean preserveIndices = Boolean.parseBoolean(System.getProperty("tests.rest.preserve_indices"));
```

Comment:
I feel like this should be a method you override and return `true` from if you want to preserve the indices from the test. That way it can be Javadoced and you can use your IDE to see who overrides it.